### PR TITLE
RSVP functionality is ready for General Interest!

### PR DESCRIPTION
**Pull request by ryanstrat**

### CODE REVIEW

File: app/Event.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    ];
+
+    public function organizer()
+    {
+        return $this->hasOne('App\User');
+    }
+
+    public function rsvp()
+    {
+        return $this->belongsTo('App\Rsvp');
```
Comment (RossIV) [inline app/Event.php]: (discussed in Slack) should be a hasMany()

File: app/User.php [hunk 2]
```diff
@@ -42,6 +42,16 @@ public function teams()
...
     }
 
+    public function organizes()
+    {
+        return $this->belongsToMany('App\Event');
+    }
+
+    public function rsvps()
+    {
+        return $this->belongsToMany('App\Rsvp');
```
Comment (RossIV) [inline app/User.php]: I think this should be hasMany as well

Comment (ryanstrat) [reply app/User.php]: Fixed

File: app/Event.php (hunk 1 above)
Comment (ryanstrat) [reply app/Event.php]: fixed

**Pull request merged**